rpcs3/Emu/Cell/SPUInterpreter.cpp

```diff
@@ -388,8 +388,19 @@ void spu_interpreter::IRET(SPUThread& spu, spu_opcode_t op)
 }
 
 void spu_interpreter::BISLED(SPUThread& spu, spu_opcode_t op)
-{
-	fmt::throw_exception("Unimplemented instruction" HERE);
+{ 
+	const v128 u =  v128::from32r(spu_branch_target(spu.pc + 4))
+	const u32 t = spu_branch_target(spu.gpr[op.ra]._u32[3]) ;
+	spu.gpr[op.rt] = u ;
+	 
+	//todo: if (/*external event*/)
+	// {		
+	// 	spu.pc = t - 4;	
+	// 	set_interrupt_status(spu, op);
+	// }
+	// else
+	spu.pc = u - 4;
+	fmt::throw_exception("TODO: SPU external event for BISLED instruction" HERE);
```

Comment:
Trowing exception at the end of partialy implemented function is wrong! This should be warning or todo.